SubjuGator/drivers/sub8_thrust_and_kill_board/sub8_thrust_and_kill_board/handle.py

```diff
@@ -108,7 +108,7 @@ def on_command(self, msg: Thrust) -> None:
         for cmd in msg.thruster_commands:
             # If we don't have a mapping for this thruster, ignore it
             if cmd.name not in self.thrusters:
-                rospy.logwarn(
+                self.get_logger().warn(
```

Comment:
I think this is going to break because `CANDeviceHandle` is not a node. I think you will need to use the global method of logging (`rclpy.logging`): https://docs.ros2.org/foxy/api/rclpy/api/logging.html